web/src/components/molecules/Content/ContentTable/index.tsx

```diff
@@ -13,15 +16,25 @@ const ContentTable: React.FC<Props> = ({ contentTableFields, contentTableColumns
     },
   };
 
-  return (
+  return contentTableColumns ? (
     <ProTable
       dataSource={contentTableFields}
-      columns={contentTableColumns}
+      columns={[
+        {
+          title: "actions",
+          render: (_, contentField) => (
+            <Link to={"./details/" + contentField.id}>
+              <Icon icon="edit" />
+            </Link>
+          ),
+        },
```

Comment:
I tried looking but couldn't figure it out. Why are you not including this column with the `contentTableColumns`?

Reply:
@KaWaite , this added object is using: 
- an atom that is the `Icon`
- the render function is returning `JSX`

`contentTableColumns` is generated in the hooks file `src/components/organisms/Project/Content/ContentList/hooks.tsx`
https://github.com/reearth/reearth-cms/blob/0e3c61829cf9937d29e61d041f60b69d02d829ce/web/src/components/organisms/Project/Content/ContentList/hooks.ts#L27-L33

So I needed a way to add this first column to the columns, without violating the above 2 points.

Reply:
It is illegal to use router in molecules though. Is there a reason you are using Link here when you seem to typically use navigate most other places? If you don't need Link for some specific reason, use navigate so you can create a callback that can be passed to the Icon. That way, you can keep atoms out of the organism and have your redirect

Reply:
@KaWaite , done, replaced the link with navigate in the organism.